samples/sqltransport/multi-instance/EndpointConnectionStringLookup/EndpointConnectionLookup.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UserImplementation
```

Comment:
you can call that out in the markdown. it will be much clearer than using a namespace